crypto/CMakeLists.txt

```diff
@@ -481,16 +481,10 @@ add_library(
   xts/xts.c
   decrepit/bio/base64_bio.c
   decrepit/blowfish/blowfish.c
-  decrepit/cast/cast.c
   decrepit/cast/cast_tables.c
   decrepit/cfb/cfb.c
-  decrepit/des/cfb64ede.c
   decrepit/dh/dh_decrepit.c
-  decrepit/dsa/dsa_decrepit.c
```

Comment:
Should we just delete these files if they're not being used anymore, or are we taking things slowly instead?
Same goes for `cast.c`, `cfb64ede.c`, `obj_decrepit.c`, `rc4_decrepit.c` and `cast_test.c`.

Reply:
Seems like git doesn't detect changes in moved files. At least this is a good data point.